src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,14 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * @param message The message to send
+     * @return The event thrown from the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
It would limit it... how? You convert to a legacy text as it is, which limits things as is.